cmd/tink-cli/cmd/workflow/list.go

```diff
@@ -27,6 +27,17 @@ var listCmd = &cobra.Command{
 	Use:     "list",
 	Short:   "list all workflows",
 	Example: "tink workflow list",
+	Deprecated: `This command is deprecated and it will change at some
```

Comment:
I don't know about the `Deprecated:` option in cobra.. is it outputted on STDOUT? My main concern is breaking people scripts?

Reply:
It prints to stdout. Personally, it is a bc break announced in this PR and in Sandbox when we will release it. I would like to remove old-style commands very soon. So, yes it breaks, mitigation is easy, do not update and I hope the `Deprecated` note and documentation will help to move fast to new commands.

Reply:
@thebsdbox I share your concern over stdout, happily though it seems cobra does the right thing and prints to stderr not stdout.
```
[~/go/src/github.com/tinkerbell/tink/cmd/tink-cli]─[manny@dellnix]> 
./tink-cli  workflow list 2>/dev/null
Exception: ./tink-cli exited with 1
[tty 24], line 1: ./tink-cli  workflow list 2>/dev/null
                                                                                                                                                            
[~/go/src/github.com/tinkerbell/tink/cmd/tink-cli]─[manny@dellnix]> 
./tink-cli  workflow list >/dev/null
Command "list" is deprecated, This command is deprecated and it will change at some
	point. Please try what follows:

	# If you want to retrieve a single workflow you know by ID
	tink workflow get [id]
	# You can print it in JSON and CSV as well
	tink workflow get -o json [id]

	# Get a list of available workflows
	tink workflow get [id]

Error: fetch cert: Get "http://127.0.0.1:42114/cert": dial tcp 127.0.0.1:42114: connect: connection refused
Usage:
  tink workflow list [flags]

Examples:
tink workflow list

Flags:
  -h, --help    help for list
  -q, --quiet   only display workflow IDs

Global Flags:
  -f, --facility string   used to build grpc and http urls

fetch cert: Get "http://127.0.0.1:42114/cert": dial tcp 127.0.0.1:42114: connect: connection refused
Exception: ./tink-cli exited with 1
[tty 25], line 1: ./tink-cli  workflow list >/dev/null
```